estate/models/estate_property.py

```diff
@@ -0,0 +1,25 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
```

Comment:
You missed this:

- Class name must match the model name (this is more vauxoo's way).

```suggestion
class EstateProperty(models.Model):
```